StepsToCare/CareDashboard.ascx.cs

```diff
@@ -179,6 +179,14 @@ namespace RockWeb.Plugins.rocks_kfs.StepsToCare
         Category = "Actions",
         Key = AttributeKey.BenevolenceDetailPage )]
 
+    [LinkedPage(
+        "Care Need History Page",
+        Description = "Page used to view history of Care Needs (if not set the action will not show)",
+        IsRequired = false,
+        Order = 11,
+        Category = "Actions",
+        Key = AttributeKey.CareNeedHistoryPage )]
```

Comment:
Ensure that the default value is added for this setting as well.

Reply:
added in https://github.com/KingdomFirst/RockAssemblies/pull/177/commits/c0dbd77af01200f49df3585a922ac770585ee9bc